Cards/MadGraph5_aMCatNLO/TT/TTto2L2Nu-3Jets_genMET-150_madgraphMLM-pythia8/TTto2L2Nu-3Jets_genMET-150_madgraphMLM-pythia8.json

```diff
@@ -0,0 +1,15 @@
+{
+    "gridpack_submit": false,
+    "gridpack_path" : "TT/TTto2L2Nu-3Jets_genMET-150_madgraphMLM-pythia8",
+    "fragment": ["Generator/ExternalLHEProducer_MadGraph5_aMCatNLO.dat", "PartonShower/madgraphMLM-pythia8-hepmcfilter.dat", "Filter/LHE_genMET150_filter.dat"],
```

Comment:
Hi Carlos. Why do these samples use the hepmc filter?

Reply:
AH you are absolutely right, I must have missed that because I copied the json from the QCD one since it was an example for reusing gridpacks. I'll change it right away